website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,64 @@
+
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when existing nodes get full, Kubernetes struggles with automatically provisioning new nodes from a cloud provider.
```

Comment:
Kubernetes isn't design to "...", it literally does "...". I'd remove the word "new" here. I think we can frame "existing" more cleanly as well.

"Kubernetes is able to pods to a set of preprovisioned nodes."